src/link.rs

```diff
@@ -40,7 +40,7 @@ fn create_project_references(children: &[String]) -> TypescriptParentProjectRefe
     let mut sorted_children = children.to_owned();
     sorted_children.sort_unstable();
     TypescriptParentProjectReference {
-        files: [].to_vec(),
```

Comment:
Note also the `refactor:` conventional-commits type https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
You're so right, I forgot about that one. I may as well change all of these to that